src/main/java/ua/javaPro/berezhnoy/hillelHomeWork6/HillelCollections.java

```diff
@@ -0,0 +1,75 @@
+package ua.javaPro.berezhnoy.hillelHomeWork6;
+
+import java.util.*;
+
+public class HillelCollections {
+    List<String> list = new ArrayList<>();
+
+    public static int countOccurance(List<String> arrList, String str) {
+        int result = Collections.frequency(arrList, str);
```

Comment:
```suggestion
       return Collections.frequency(arrList, str);
```
можна одразу повертати.